composer.json

```diff
@@ -18,8 +18,7 @@
     "require": {
         "neos/neos": "@dev",
         "neos/site-kickstarter": "@dev",
-
-        "neos/demo": "@dev",
+        
         "neos/neos-ui": "@dev",
         "neos/seo": "@dev",
```

Comment:
Not sure wether Neos Seo should be required here or via Neos.Demo

Reply:
Yeah, up for debate, same could be said for the redirect packages.

Reply:
I agree, let us discuss this topic once after this was merged

Reply:
I would like to re-open that discussion because that was exactly my thought.
Of course I'm totally fine with changing that afterwards, but IMO the name "empty Neos" should be changed because it's probably not a good idea to rename that once this is released